src/lib/viewers/controls/annotations/AnnotationsControls.tsx

```diff
@@ -33,12 +34,27 @@ export default function AnnotationsControls({
     const showDrawing = !isFullscreen && hasDrawing;
     const showHighlight = !isFullscreen && hasHighlight;
     const showRegion = !isFullscreen && hasRegion;
+    const annotationBtnRefs = {
```

Comment:
Minor, but can we alphabetize the variables here? We can also skip the abbreviation via `buttonRefs` or something.